docs/.readthedocs.yaml

```diff
@@ -0,0 +1,19 @@
+# .readthedocs.yml
```

Comment:
From looking at the [latest run](https://readthedocs.org/projects/flyte/builds/22743753/), `readthedocs` is not picking this configuration up and is using the top level configuration.

Should this this be moved up and override the original configuration?

Reply:
we're planning on the monodocs site here: https://flyte-next.readthedocs.io/en/latest/introduction.html to manage the transition.

once we're confident about all the changes we'll make this the default configuration

Reply:
see here for more details: https://unionai.slack.com/archives/C01R4HFGR2T/p1700700721963309